FIO compressed tests are failing with "Exception: fio-client-13b7da5d--1-m9jtj Failed to run !!!" error

FIO compressed tests are failing to start FIO pods and hence resulting in test failure

Following stack trace is seen:
            for name in results:
                # looking for the pod which run the benchmark (not the IO)
                # this pod contain the `client` in his name, and there is only one
                # pod like this, other pods have the `server` in the name.
                (fname, status) = name.split()
                if re.search("client", fname):
                    break
                else:
                    (fname, status) = ["", ""]
    
            if fname == "":  # there is no `client` pod !
                err_msg = f"{self.client_pod} Failed to run !!!"
                log.error(err_msg)
>               raise Exception(err_msg)
E               Exception: fio-client-13b7da5d--1-m9jtj Failed to run !!!

--- a/ocs_ci/templates/workloads/fio/benchmark_fio_cmp.yaml
+++ b/ocs_ci/templates/workloads/fio/benchmark_fio_cmp.yaml
@@ -31,6 +31,7 @@ spec:
       storageclass: ocs-storagecluster-ceph-rbd-cmp
       storagesize: 12Gi
       cmp_ratio: 75
+      job_timeout: 18000
 #      rook_ceph_drop_caches: True
 #      rook_ceph_drop_cache_pod_ip:
 #######################################

--- a/tests/e2e/performance/io_workload/test_fio_benchmark.py
+++ b/tests/e2e/performance/io_workload/test_fio_benchmark.py
@@ -21,8 +21,6 @@ from ocs_ci.ocs import benchmark_operator
 
 log = logging.getLogger(__name__)
 
-SKIP_REASON = "Skipping FIO compressed test until github issue 5099 is addressed"
-
 
 class FIOResultsAnalyse(PerfResult):
     """
@@ -455,7 +453,6 @@ class TestFIOBenchmark(PASTest):
             log.info(f"The Result can be found at : {full_results.results_link()}")
 
     @skipif_ocs_version("<4.6")
-    @pytest.mark.skip(SKIP_REASON)
     @pytest.mark.parametrize(
         argnames=["io_pattern", "bs", "cmp_ratio"],
         argvalues=[

